### ISSUE TITLE
Ledger cache is probably not working as expected

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#2221]
If you log the ledger node cache on line 409 of `Splash.tsx` it appers to contain defunct nodes when tested with `nmap` or `nc`. I think the idea behind the cache is it is mean to cache the reconsiled (know to be good) nodes for a given ledger:

```typescript
setStep(4)
const cachedLedgers = await loadCachedLedgers()
console.log('************** cached ledgers = ', JSON.stringify(cachedLedgers))
const ledgers = cachedLedgers ?? indyLedgers
```

Taking two node IPs from the output and testing them with `nc` shows one to be working and another to be down. If the list was reconciled then one would expect all nodes to be active.

### Failed Node on sovrin:test

```console
➜  core git:(main) nc -zv -G 5 52.64.96.160 9702
nc: connectx to 52.64.96.160 port 9702 (tcp) failed: Operation timed out
```

### Active Node on bcovrin:test

```console
➜  core git:(main) nc -zv -G 5 138.197.138.255 9702                
Connection to 138.197.138.255 port 9702 [tcp/*] succeeded!
```

**Comment by jleach** [comment]
I don't think we need to action this one. We'll soon cut over to the Indy-VDR proxy and that will make this a non-issue. It can be closed when this happens (1.0.23 or .24).

**Comment by jeznorth** [comment]
@jleach can you confirm this is still an issue?
